app/uk/gov/hmrc/hec/models/SAStatus.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hec.models
+
+import julienrf.json.derived
+import play.api.libs.json.OFormat
+
+sealed trait SAStatus {
+  val IFString: String
+}
+object SAStatus {
+  case object ReturnFound extends SAStatus {
```

Comment:
`CTStatus` has one extra status `NoAccountingPeriodFound`, so we can't reuse I think unless we use a library like [Enumeratum](https://github.com/lloydmeta/enumeratum which would allow us to do sub-groups of statuses

Reply:
You can create a sealed trait like Status and then something like this
case object ReturnFound extends ReturnStatus { override def value = "Return Found" }
  case object NoticeToFileIssued extends ReturnStatus { override def value = "Notice to File Issued" }
  case object NoReturnFound extends ReturnStatus { override def value = "No Return Found" }
  case object NoAccountingPeriodFound extends ReturnStatus { override def value = "No Accounting Period Found" }
  
  I don't think it will cause problem

Reply:
it's a bit annoying to have `NoAccountingPeriodFound` for SA when we don't need it cos we'll always have to cater for that one case that's never gonna get triggered when we pattern match over it. With two sealed traits it allows flexibility to change independently in the future as well for not too much of a price to pay